apiserver/facades/agent/uniter/networkinfo.go

```diff
@@ -97,11 +102,11 @@ func NewNetworkInfoForStrategy(
 	defaultSpaceID := network.AlphaSpaceId
 	defaultSpaceName := cfg.DefaultSpace()
 	if defaultSpaceName != "" && defaultSpaceName != network.AlphaSpaceName {
-		defaultSpace, err := st.SpaceByName(defaultSpaceName)
+		defaultSpace, err := networkService.SpaceByName(ctx, defaultSpaceName)
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		defaultSpaceID = defaultSpace.Id()
+		defaultSpaceID = defaultSpace.ID
```

Comment:
Goodness me. I probably wrote this, but it doesn't seem worth the trouble. We should just use a constant for the default space ID.